modules/machineset-yaml-azure.adoc

```diff
@@ -71,19 +110,46 @@ spec:
           resourceGroup: <infrastructureID>-rg <1>
           sshPrivateKey: ""
           sshPublicKey: ""
+ifndef::infra[]
           subnet: <infrastructureID>-<role>-subnet <1> <2>
           userDataSecret:
             name: <role>-user-data <2>
+endif::infra[]
+ifdef::infra[]
+          subnet: <infrastructureID>-infra-subnet <1> <2>
```

Comment:
the subnet value is not correct, it should be something like` qeci-22538-worker-subnet`, you can obtain the subnet by running the following command:
```
  $  oc -n openshift-machine-api \
    -o jsonpath='{.spec.template.spec.providerSpec.value.subnet}{"\n"}' \
    get machineset/<infrastructureID>-worker-centralus1
```